smarts/core/tests/test_renderers.py

```diff
@@ -35,22 +36,51 @@
 from smarts.core.scenario import Scenario
 from smarts.core.smarts import SMARTS
 from smarts.core.sumo_traffic_simulation import SumoTrafficSimulation
+from smarts.core.utils.config import SmartsConfig
 from smarts.core.vehicle import RendererException
 
 
+def _smarts_with_agent(request, agent) -> SMARTS:
```

Comment:
I don't see where the `request` parameter gets used here (or anywhere else below).

Reply:
I gets added through the command line from the local `conftest.py`.